docs/data-sources/l3out_consumer_label.md

```diff
@@ -0,0 +1,60 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+subcategory: "L3Out"
+layout: "aci"
+page_title: "ACI: aci_l3out_consumer_label"
+sidebar_current: "docs-aci-data-source-aci_l3out_consumer_label"
+description: |-
+  Data source for L3out Consumer Label
+---
+
+# aci_l3out_consumer_label #
+
+Data source for L3out Consumer Label
+
+## API Information ##
+
+* Class: [l3extConsLbl](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extConsLbl/overview)
+
+* Supported in ACI versions: 2.0(1m) and later.
+
+* Distinguished Name Formats
+  - `uni/tn-{name}/out-{name}/conslbl-{name}`
+
+## GUI Information ##
+
+* Location: Tenants -> Networking -> SR MPLS VRF L3Outs -> SR-MPLS Infra L3Outs
+
+## Example Usage ##
+
+```hcl
+
+data "aci_l3out_consumer_label" "example_l3_outside" {
+  parent_dn = aci_l3_outside.example.id
+  name      = "test_name"
+}
+
+```
+
+## Schema
+
+### Required
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_l3_outside](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/l3_outside) ([l3extOut](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extOut/overview))
+* `name` (name) - (string) The name of the L3out Consumer Label object.
+
+### Read-Only
+
+* `id` - (string) The distinguished name (DN) of the L3out Consumer Label object.
+* `annotation` (annotation) - (string) The annotation of the L3out Consumer Label object.
+* `description` (descr) - (string) The description of the L3out Consumer Label object.
+* `name_alias` (nameAlias) - (string) The name alias of the L3out Consumer Label object.
+* `owner` (owner) - (string) The owner of the L3out Consumer Label object.
```

Comment:
should we use scope like in Ansible?

Reply:
there is no scope defined for this in ansible, confusion with dhcp relay module